Library/Homebrew/utils/analytics.rb

```diff
@@ -16,64 +16,53 @@ class << self
 
       include Context
 
-      def report(type, metadata = {})
+      BUCKET = "analytics"
+      TOKEN = "bHDUBcO8rp521Y5lzCV4nrhWaYOeV-OjGeDVxv37m9Ee75CwTjir4EQr1hCHNJaEeRowiAjw3uSh9jG1Usmqfw=="
+      HOST = "europe-west1-1.gcp.cloud2.influxdata.com"
+
+      def report(category, metadata = {})
         return if not_this_run?
         return if disabled?
 
-        analytics_ids = ENV.fetch("HOMEBREW_ANALYTICS_IDS", "").split(",")
-        analytics_ids.each do |analytics_id|
-          args = []
-
-          # do not load .curlrc unless requested (must be the first argument)
-          args << "--disable" unless Homebrew::EnvConfig.curlrc?
-
-          args += %W[
-            --max-time 3
-            --user-agent #{HOMEBREW_USER_AGENT_CURL}
-            --data v=1
-            --data aip=1
-            --data t=#{type}
-            --data tid=#{analytics_id}
-            --data cid=#{ENV["HOMEBREW_ANALYTICS_USER_UUID"]}
-            --data an=#{HOMEBREW_PRODUCT}
-            --data av=#{HOMEBREW_VERSION}
-          ]
-          metadata.each do |key, value|
-            next unless key
-            next unless value
-
-            key = ERB::Util.url_encode key
-            value = ERB::Util.url_encode value
-            args << "--data" << "#{key}=#{value}"
-          end
+        args = []
+        fields = []
 
-          # Send analytics. Don't send or store any personally identifiable information.
-          # https://docs.brew.sh/Analytics
-          # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
-          # https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters
-          if ENV["HOMEBREW_ANALYTICS_DEBUG"]
-            url = "https://www.google-analytics.com/debug/collect"
-            puts "#{ENV["HOMEBREW_CURL"]} #{args.join(" ")} #{url}"
-            puts Utils.popen_read ENV["HOMEBREW_CURL"], *args, url
-          else
-            pid = fork do
-              exec ENV["HOMEBREW_CURL"],
-                   *args,
-                   "--silent", "--output", "/dev/null",
-                   "https://www.google-analytics.com/collect"
-            end
-            Process.detach T.must(pid)
+        # do not load .curlrc unless requested (must be the first argument)
+        args << "--disable" unless Homebrew::EnvConfig.curlrc?
+
+        metadata.each do |key, value|
+          next unless key
+          next unless value
+
+          fields << "#{key}=#{value}"
+        end
+
+
+        args += %W[
+          --max-time 3
+          --header "Content-Type: text/plain; charset=utf-8"
+          --header "Accept: application/json"
+          --header "Authorization: Token #{TOKEN}"
+          --data-raw '#{category},version=#{HOMEBREW_VERSION},product=#{HOMEBREW_PRODUCT},#{fields.join(',')} count=1i #{Time.now.to_i}'
+        ]
+
+        url = "https://#{HOST}/api/v2/write?bucket=#{BUCKET}&precision=s"
```

Comment:
We'll want to report to both GA and InfluxDB for a while (minimum 30 days) until we've got adequate data to switch over.